File: services/flashcard/src/main/kotlin/si/um/kotnik/application/dtos/CardSetDto.kt
package si.um.kotnik.application.dtos

data class CardSetDto(
    var id: String = "",
    var name: String = "",
    var userEmail: String = "",
    var languageName: String = "",
    var languageCode: String = "",
    var flashCards: List<FlashCardDto> = emptyList(),
)

File: bff/web/WebBFF/Application/Dtos/Cards/CardSetDto.cs
namespace WebBFF.Application.Dtos.Cards;

public class CardSetDto
{
    public string Id { get; set; } = string.Empty;
    public string Name { get; set; } = string.Empty;
    public string UserEmail { get; set; } = string.Empty;
    public string LanguageName { get; set; } = string.Empty;
    public string LanguageCode { get; set; } = string.Empty;
    public List<FlashCardDto> Flashcards { get; set; } = new();

}